examples/pong_tutorial_01/main.rs

```diff
@@ -0,0 +1,56 @@
+extern crate amethyst;
+
+use amethyst::prelude::*;
+use amethyst::renderer::{DisplayConfig, DrawFlat, Event, KeyboardInput, Pipeline, PosNormTex,
+                         RenderBundle, RenderSystem, Stage, VirtualKeyCode, WindowEvent};
+
+struct Pong;
+
+impl State for Pong {
+    fn handle_event(&mut self, _: &mut World, event: Event) -> Trans {
+        match event {
+            Event::WindowEvent { event, .. } => {
+                match event {
+                    WindowEvent::KeyboardInput {
+                    input:
+                        KeyboardInput {
+                            virtual_keycode: Some(VirtualKeyCode::Escape),
+                            ..
+                        },
+                    ..
+                } => Trans::Quit,
+                    _ => Trans::None,
+                }
+            }
+            _ => Trans::None,
+        }
+    }
+}
+
+fn run() -> Result<(), amethyst::Error> {
+    let path = format!("{}/examples/pong_tutorial_01/resources/display_config.ron",
+                       env!("CARGO_MANIFEST_DIR"));
+    let config = DisplayConfig::load(&path);
+
+    let pipe = Pipeline::build().with_stage(Stage::with_backbuffer()
+        .clear_target([0.0, 0.0, 0.0, 1.0], 1.0)
+        .with_pass(DrawFlat::<PosNormTex>::new()));
+
+    let mut game = Application::build("./", Pong)
+        ?
+        .with_bundle(RenderBundle::new())?
+        .with_local(RenderSystem::build(pipe, Some(config))?)
+        .build()
+        .expect("Fatal error");
```

Comment:
As noted above, this should return the errror

Reply:
Replace the expect with `?`

Reply:
Yeah